plugins/woocommerce/src/StoreApi/Utilities/QuantityLimits.php

```diff
@@ -50,9 +50,9 @@ public function get_cart_item_quantity_limits( $cart_item ) {
 	 * @return array
 	 */
 	public function get_add_to_cart_limits( \WC_Product $product ) {
-		$multiple_of = $this->filter_value( 1, 'multiple_of', $product );
-		$minimum     = $this->filter_value( 1, 'minimum', $product );
-		$maximum     = $this->filter_value( $this->get_product_quantity_limit( $product ), 'maximum', $product );
+		$multiple_of = (int) $this->filter_value( 1, 'multiple_of', $product );
+		$minimum     = (int) $this->filter_value( 1, 'minimum', $product );
+		$maximum     = (int) $this->filter_value( $this->get_product_quantity_limit( $product ), 'maximum', $product );
```

Comment:
This only casts, but does it check if someone returned I dunno a random abc string? This would evaluate to 0, which works fine for minimum but would break multiple and maximum

Reply:
This is true if the filter is misused. The filter docblock and passed value is int. The error logs showed that someone filtered it to return `''` empty string. I guess we could add another layer of value forcing.